Common.md

```diff
@@ -20,3 +40,130 @@ Language names are title cased.
 | [dropwizard](dropwizard) | `DropWizard` | `NewRelic-DropWizard-Exporter/0.1.0` |
 
 A full `User-Agent` example might look like `NewRelic-Python-TelemetrySDK/0.2.3 NewRelic-Python-OpenCensus/0.1.0`.
+
+## Attributes
+
+The origins of telemetry exported to New Relic needs to be identifiable in order for it to be useful.
+Being able to identify what generated, measured, collected, and transmitted telemetry to New Relic is essential to providing a curated customer experience.
+This section describes the attributes are used to identify these dimensions of the telemetry origins.
+
+The following table is an overview for quick reference of all the attributes defined in the following sub-sections.
+
+| Attribute | REQUIRED/RECOMMENDED/OPTIONAL |
+| --------- | ----------------------------- |
+| `instrumentation.provider` | REQUIRED |
+| `instrumentation.name` | RECOMMENDED |
+| `instrumentation.version` | RECOMMENDED |
+| `collector.name` | RECOMMENDED |
+| `collector.version` | RECOMMENDED |
+| `host.hostname` | REQUIRED† |
+| `service.name` | REQUIRED† |
+| `app.name` | REQUIRED† |
+| `db.name` | REQUIRED† |
+
+†: REQUIRED only if annotating an applicable telemetry source.
+See the related sub-section for more information.
+
+### Instrumentation
+
+Instrumentation is the toolset used to measure telemetry.
+This instrumentation usually comes in the form of a library of code, platform, or coding language builtin and provides different instruments to measure the state of a computer program.
+This instrumentation is from what the exporter exports.
+
+Including attributes to identify not only what instrumentation is used but also what code is being instrumented is crucial for interoperability between exporters and the New Relic platform.
+
+#### `instrumentation.provider`
+
+All exported telemetry MUST contain an `instrumentation.provider` attribute that identifies the provider of instrumentation.
+
+Example values of this attribute are:
+
+* `newRelic`
+* `prometheus`
+* `dropwizard`
+* `opencensus`
+* `opentelemetry`
+
+#### `instrumentation.name`
+
+All exported telemetry SHOULD contain an `instrumentation.name` attribute that identifies the code that is being instrumented.
+This would be the framework, library, or application that uses the instrumentation to report telemetry.
```

Comment:
It might be worth noting that this should only be reported if it's actually available via the instrumentation provider. If it's not, then I think it should be left out. Same with instrumentation.version.